build.sbt

```diff
@@ -49,6 +53,8 @@ libraryDependencies ++= Seq(
   "eus.ixa" % "ixa-pipe-nerc" % "1.6.1"
 )
 
+libraryDependencies += "nz.ac.waikato.cms.weka" % "optics_dbScan" % "1.0.4"
```

Comment:
You made some visualizations of the fortis-v1 data, right? I remember your post about plotly. Could you share the visualizations with the group so that we can brainstorm about what sorts of assumptions we can and cannot make about the data?